### DISCUSSION TITLE
Freezing when syncing

### DISCUSSION THREAD

**Discussion by aKaiser79** [javikr/sofatime#24] [Ideas]
It would be nice if the Trakt sync didn't interfere with the haptics of the app.
When you start the app, it often freezes for a few seconds. This is not a nice user experience, maybe this can be run in the background? Or disable the check buttons for the time of sync if this is done to safeguard against sync errors.

**Comment by javikr** [comment]
Hi!
I'm agree with you. Trakt sync often takes few seconds to finish, it depends on the data that needs to load. The syncing process itself is made in a background thread, but the UI refresh is made in the main thread, this is the reason why sometimes the app freezes.

I'm still working on a solution here and I will take in account your approach ;)
